src/engraving/dom/cmd.cpp

```diff
@@ -3034,7 +3037,7 @@ EngravingItem* Score::move(const String& cmd)
 
         // Skip gap rests if we're not in note entry mode...
         while (!noteEntryMode() && el && el->isRest() && toRest(el)->isGap()) {
-            el = prevChordRest(toChordRest(el));
+            el = noteEntryPos ? el : prevChordRest(toChordRest(el));
```

Comment:
If `noteEntryPos` is true, then this reads `while (…) { el = el; }`, so how are we not ending up in an infinite loop?

Reply:
@worldwideweary ?

Reply:
@Jojo-Schmitz The original code from my PR to 3.7 didn't have this problem @ https://github.com/Jojo-Schmitz/MuseScore/pull/605/files (the assignment was outside of the while loop ;) plus it could just be an if statement if the ternary feels awkward like `if (!noteEntryPosition) el = prevChordRest(cr)`

Reply:
![image](https://github.com/user-attachments/assets/345320c5-08a1-4c86-b812-2be06b374b87)

Reply:
Oops, porting error...